ding/model/common/head.py

```diff
@@ -478,7 +476,8 @@ def forward(self, x: torch.Tensor) -> Dict:
 class QuantileHead(nn.Module):
     """
     Overview:
-        The ``QuantileHead`` used to output action quantiles, which is used in IQN.
+        The ``QuantileHead`` is used to output action quantiles.
```

Comment:
add some detailed description about the difference betweene QRDQN/IQN/FQF's head